kevm-pyk/src/kevm_pyk/kdist/plugin.py

```diff
@@ -27,17 +27,20 @@ def __init__(self, kompile_args: Mapping[str, Any], *, deps: Iterable[str] | Non
     def build(self, output_dir: Path, deps: dict[str, Path], args: dict[str, Any]) -> None:
         verbose = args.get('verbose', False)
         enable_llvm_debug = args.get('enable_llvm_debug', False)
+        debug_build = args.get('debug_build', False)
+        ccopts = args.get('ccopts', [])
 
         kevm_kompile(
             output_dir=output_dir,
             enable_llvm_debug=enable_llvm_debug,
             verbose=verbose,
-            plugin_dir=deps.get('evm-semantics.plugin'),
+            ccopts=ccopts,
+            debug_build=debug_build,
             **self._kompile_args,
         )
 
     def deps(self) -> tuple[str, ...]:
-        return self._deps
+        return tuple(unique(['evm-semantics.plugin'] + list(self._deps)))
```

Comment:
Consider

```suggestion
        return ('evm-semantics.plugin',)
```

and dropping `self._deps` altogether.

Reply:
Done.